app/src/main/java/org/fossasia/openevent/app/core/settings/SettingsFragment.java

```diff
@@ -20,6 +21,9 @@
 public class SettingsFragment extends PreferenceFragmentCompat {
 
     private static final String VERSION = "Version";
+    private static final String GROSS_SALES = "Gross Sales";
+    private static final String NET_SALES = "Net Sales";
+    private PreferenceManager manager;
```

Comment:
Can be final?

Reply:
It is initialized in onCreatePreferencesFix. Hence, cannot be final.